streamlit_pianoroll/frontend/public/styles.css

```diff
@@ -3,15 +3,24 @@
   padding: 0 !important;
 }
 
-.row {
-  margin: 0 !important;
-}
-
 .pianoroll-svg-wrapper {
   position: relative;
   border: 3px solid black;
   padding: 0px !important;
   background-color: #f0f0f0;
+  height: 100%;
+}
+
+.pianoroll-progress-bar-wrapper {
+  border: 3px solid black;
+  border-top: none;
+  cursor: pointer;
+}
+
+.pianoroll-progress-bar {
+  background-color: whitesmoke;
+  width: 100%;
+  height: 6.25rem;
```

Comment:
Can we make this relative to the upper svg height? In the standard streamlit view it looks great for me with `height: 3.00rem;`:

<img width="420" alt="image" src="https://github.com/Nospoko/streamlit-pianoroll/assets/8056825/9464698e-f88e-404b-b7d1-5da07d2b2ff6">

But in the "wide mode" I think 5rem looks better and is similar in ratio:

<img width="420" alt="image" src="https://github.com/Nospoko/streamlit-pianoroll/assets/8056825/165eac9e-5b09-4c9a-bcfa-4a6d3abebfb8">